cilium-dbg/cmd/service_update.go

```diff
@@ -119,6 +120,11 @@ func updateService(cmd *cobra.Command, args []string) {
 	warnIdTypeDeprecation()
 
 	id := int64(idU)
+	maxID := int64(service.MaxSetOfServiceID - 1)
+	if id > maxID {
```

Comment:
`int64(service.MaxSetOfServiceID - 1)` is 65534. So this code doesn't accept id 65535. Is that correct?

Reply:
maxID is incorrect, it should be 65535.

I have updated the code. Thanks for your review.